docs/config/README.md

```diff
@@ -794,6 +795,31 @@ symbol = "+ "
 threshold = 4
 ```
 
+## Julia
+
+The `julia` module shows the currently installed version of Julia.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `Project.toml` file
+- The current directory contains a `Manifest.toml` file
+- The current directory contains a file with the `.jl` extension
+
+### Options
+
+| Variable   | Default       | Description                                              |
+| ---------- | ------------- | -------------------------------------------------------- |
+| `symbol`   | `"∴ "`       | The symbol used before displaying the version of Julia. |
+| `style`    | `"bold purple"` | The style for the module.                                |
+| `disabled` | `false`       | Disables the `julia` module.                            |
+
+### Example
+
+```toml
+# ~/.config/starship.toml
+
+[julia]
+symbol = "👸 "
```

Comment:
I believe this example may not be compliant with [Julia's community standards](https://julialang.org/community/standards/):

> do not sexualize the term "Julia" or any other aspects of the project. While "Julia" is a female name in many parts of the world, the programming language is not a person and does not have a gender.

I'd suggest using something different, e.g. "Mathematical Bold Capital J" (U+1D409):

    symbol = "𝐉 "

or "Mathematical Sans-serif Bold Small J" (U+1D5F7), which looks similar to the j in [Julia's logo](https://julialang.org):

    symbol = "𝗷 "

WDYT?

Reply:
Agreed, it would probably be best if we changed it.
I like both your suggestions, @waldyrious. Another option if the "Therefore" symbol, present in the Julia logotype and favicon: `∴` or `ஃ`

Reply:
The therefore symbol is actually already the default :) if I understood the code correctly, this was an example of customizing the symbol. However, the second symbol you listed is an even better match to the Julia logo! I'd suggest replacing the current default with it and using one of the J symbols I proposed as an example of customization.

Reply:
> The therefore symbol is actually already the default :)

Ah right. My bad. 🤦‍♂
This is what happens when I swoop into issues without reading the context. 😅 

> However, the second symbol you listed is an even better match to the Julia logo! I'd suggest replacing the current default with it and using one of the J symbols I proposed as an example of customization.

Sounds good to me. 👍 
Would you be open to creating a PR for the change, Waldir?

Reply:
Oh.. I didn't know https://julialang.org/community/standards/ :bow:

I will fix this as soon as possible.